x-pack/plugins/security_solution/public/common/containers/source/use_data_view.tsx

```diff
@@ -148,7 +146,7 @@ export const useDataView = (): {
                   endTracking('success');
 
                   const patternString = response.indicesExist.sort().join();
-                  if (needToBeInit && scopeId) {
+                  if (needToBeInit && scopeId && !skipScopeUpdate) {
```

Comment:
If the data view needs to be init (get fields/pattern list) but the sourcerer scope has a saved pattern list (timeline), then do not update the selected data view